docs/docs/migration/migration.md

```diff
@@ -0,0 +1,21 @@
+# Migration from v1
+
+:::danger
+Your workload may begin processing from the earliest data due to improper migration from KP v1 to v2 or from any other kakfa framework, which could result in processing the same data twice.
+:::
+
+:::warning
+KP v1 adds `-kp` as suffix to the group name so if you are using the same name you will be required to add `-kp` back
+:::
+
+The name of the consumer group must remain unchanged, which is the most crucial point to remember. To prevent any problems, be sure to copy the consumer group name from an admin UI.
```

Comment:
might be helpful to provide a brief explanation about why it's crucial, like it would cause the consumers to reconsume all the old messages in the queue.